Source/Core/Core/HW/GCPadEmu.h

```diff
@@ -35,7 +35,7 @@ class GCPad : public ControllerEmu::EmulatedController
 {
 public:
   explicit GCPad(unsigned int index);
-  GCPadStatus GetInput() const;
+  GCPadStatus GetInput();
```

Comment:
Any particular reason why this isn't `const` anymore?

Reply:
Mistake